runtime/common/src/migrations.rs

```diff
@@ -194,6 +246,7 @@ where
 			// Box::new(remove_collectives_addresses),
 			// permanent migrations
 			Box::new(MigrateToLatestXcmVersion::<Runtime>(Default::default())),
+			Box::new(MigrateCodeToStateTrieV1::<Runtime>(Default::default())),
```

Comment:
This should not be considered a permanent migration

Reply:
Didn't notice the comment separation .

Reply:
Fixed